pkgs/development/python-modules/manuf/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, runCommand
+, python3
+, wireshark-cli
+}:
+
+buildPythonPackage rec {
+  pname = "manuf";
+  version = "1.1.5";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "coolbho3k";
+    repo = "manuf";
+    rev = "${version}";
+    hash = "sha256-3CFs3aqwE8rZPwU1QBqAGxNHT5jg7ymG12yBD56gTNI=";
+  };
+
+  nativeBuildInputs = [ wireshark-cli ];
+
+  prePatch = ''
+    cat ${wireshark-cli}/share/wireshark/{manuf,wka} > manuf/manuf
+  '';
+
+  patches = [ ./internal_db_update_nix.patch ];
+
+  checkPhase = ''
```

Comment:
I think we can use `pytestCheckHook` here.

Reply:
Done